tests/test_black.py

```diff
@@ -1,12 +1,17 @@
 # -*- coding: utf-8 -*-
 
+import pytest
+
+
+pytestmark = pytest.mark.usefixtures('black_available')
+
 
 def test_help_message(testdir):
     result = testdir.runpytest("--help")
     result.stdout.fnmatch_lines(["*--black*enable format checking with black"])
 
 
-def test_fail(testdir):
+def test_fail(testdir, black_available):
```

Comment:
do you still need this for each individual test if you already set it up for the module above?

Reply:
It should not be needed. That was the intention by adding the mark. Fixed in ccee75d.